src/main/java/mate/academy/dao/MovieSessionDao.java

```diff
@@ -0,0 +1,17 @@
+package mate.academy.dao;
+
+import java.time.LocalDateTime;
+import java.util.List;
+import java.util.Optional;
+import mate.academy.model.MovieSession;
+
+public interface MovieSessionDao {
+    MovieSession add(MovieSession movieSession);
+
+    Optional<MovieSession> get(Long id);
+
+    List<MovieSession> getAll();
+
+    List<MovieSession> findAvailableSessions(Long movieId, LocalDateTime start, LocalDateTime end);
+
```

Comment:
remove the last empty line in all classes/interfaces/methods where it is present
```suggestion
```